app/src/main/res/resource/values-de/strings.xml

```diff
@@ -223,21 +225,21 @@
     <string name="action_align_right">Rechts anheften</string>
```

Comment:
This "anheften" stuff is usually translated as "ausrichten". Just look in your favourite word processing program.
Also "Zentriert ausrichten".